pkg/cmd/server/api/v1/conversions.go

```diff
@@ -219,6 +219,15 @@ func addDefaultingFuncs(scheme *runtime.Scheme) error {
 				obj.ServiceAccountMethod = "prompt"
 			}
 		},
+		func(obj *AdmissionConfig) {
```

Comment:
> Why does this live in conversion and not in defaulting?

Put another way, even if we didn't convert this, you'd still want to end up with non-nil values in the external object, right?